plugins/graphcms.js

```diff
@@ -9,5 +9,8 @@ export default (
       nuxtState.nuxtState.config.graphcms_endpoint = graphcms_endpoint;
     });
   }
-  inject("graphcms", new GraphQLClient(graphcms_endpoint));
+  inject(
+    "graphcms",
+    graphcms_endpoint ? new GraphQLClient(graphcms_endpoint) : graphcms_endpoint

```

Comment:
Clearer when explicitly assign the variabler as `graphcms_endpoint ? new GraphQLClient(graphcms_endpoint) : undefined`.